runtime/onert/api/nnfw/src/nnfw_session.cc

```diff
@@ -101,16 +101,16 @@ NNFW_STATUS getTensorIndexImpl(const onert::ir::IGraph &graph, const char *tenso
   }
 }
 
-std::string trim(const std::string &value)
+std::string trim(std::string_view value)
 {
-  std::string whitespace = " \t";
+  constexpr std::string_view whitespace = " \t";
```

Comment:
In C++20, many member functions of `std::string_view` (including `find_first_not_of`) are declared as `constexpr` and can be evaluated at compile time if given constant expressions. However, for `std::string`, prior to C++20 its member functions (including `find_first_not_of`) are not `constexpr` and thus cannot be used as compile-time constants. In short, `find_first_not_of` can be a compile‐time constant when used with types that support `constexpr` evaluation (like `std::string_view` in C++20), but not in general.